opentok.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   # spec.add_development_dependency "debugger", "~> 1.6.6"
 
   spec.add_dependency "addressable", "~> 2.3" #  2.3.0 <= version < 3.0.0
-  spec.add_dependency "httparty", "~> 0.14.0"
+  spec.add_dependency "httparty", ">= 0.14.0"
```

Comment:
@Lucashuang0802 Why not set it to `~> 0.15.5`?

Reply:
Hey, can we get any activity on this?

Reply:
@aiham 0.15.5 requires ruby v2 or higher